csrc/host_ir/pass/deallocate.cpp

```diff
@@ -0,0 +1,52 @@
+// clang-format off
+/*
+ * SPDX-FileCopyrightText: Copyright (c) 2025-present NVIDIA CORPORATION & AFFILIATES.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+// clang-format on
+
+#include <host_ir/pass/deallocate.h>
+
+namespace nvfuser::hir {
+
+void insertDeallocations(HostIrContainer* hic) {
+  const std::vector<Expr*>& topLevelExprs = hic->topLevelExprs();
+  std::unordered_map<TensorView*, std::vector<nvfuser::Expr*>::size_type>
```

Comment:
```suggestion
  std::unordered_map<TensorView*, int64_t>
```

Reply:
Thanks, I need to keep this c++ guideline in mind